### PR TITLE
hide health bar when full or empty.

### LINKED ISSUE #156: Health bar
- [x] Hide when dead
- [x] Hide when 100% HP

### PR DESCRIPTION
**Pull request by yuenhol**
hide health bar when full or empty.
Also always show health bar for player on the top right

### CODE REVIEW

File: Assets/Game/ScriptableObjects/Spawn by Shortcut.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```
Comment (makkamchuen) [inline Assets/Game/ScriptableObjects/Spawn by Shortcut.meta]: Is this file needed? the file name seems create by accident

**Review by makkamchuen — CHANGES_REQUESTED** [review]

File: Assets/Game/ScriptableObjects/Spawn by Shortcut.meta (hunk 1 above)
Comment (yuenhol) [reply Assets/Game/ScriptableObjects/Spawn by Shortcut.meta]: Liked many meta files, this file might just modified by unity by itself when I open or close the game on Unity.

**Review by makkamchuen — APPROVED** [review]
